src/Infrastructure.EntityFramework/AdminConsole/Repositories/OrganizationUserRepository.cs

```diff
@@ -721,4 +721,14 @@ public UpdateEncryptedDataForKeyRotation UpdateForKeyRotation(
             return data;
         }
     }
+
+    public async Task SetOrganizationUsersStatusAsync(IEnumerable<Guid> userIds, OrganizationUserStatusType status)
+    {
+        using var scope = ServiceScopeFactory.CreateScope();
+
+        var dbContext = GetDatabaseContext(scope);
+
+        await dbContext.OrganizationUsers.Where(x => userIds.Contains(x.OrganizationId))
+            .ExecuteUpdateAsync(s => s.SetProperty(x => x.Status, status));
```

Comment:
You also need to bump the account revision dates here. See https://github.com/bitwarden/server/blob/e16cad50b16a8bed011f86c039e6892c4ebfe9cf/src/Infrastructure.EntityFramework/Repositories/DatabaseContextExtensions.cs#L122-L123.